### PR TITLE
continue project from MyProfile-Involvments-Clutter - resolved merge conflicts

### PR DESCRIPTION
**Pull request by antoniavonto**
Trying to continue the work from the pull request https://github.com/gordon-cs/gordon-360-ui/pull/1083

Most code was taken from that pull request, but some had to change because of merge conflicts. 

Though it would probably would have been better to merge develop to that pull request and resolve merge conflicts, I thought a new branch that I could add the code in logical order was easier for me to understand and make the necessary changes to solve the conflicts.

see: https://github.com/gordon-cs/gordon-360-api/pull/934

### CODE REVIEW

**Comment by antoniavonto** [comment]
image

**Comment by antoniavonto** [comment]
@amos-cha

File: src/components/Profile/components/MembershipsList/components/MembershipInfoCard/MembershipInfoCard.module.scss [hunk 1]
```diff
@@ -40,7 +41,7 @@
   margin-right: 0.5rem;
 }
 
-@media (max-width: 500px) {
+@media (max-width: 700px) {
```
Comment (amos-cha) [inline src/components/Profile/components/MembershipsList/components/MembershipInfoCard/MembershipInfoCard.module.scss:44]:
```suggestion
@media (max-width: $break-sm) {
```

File: src/components/Profile/components/MembershipsList/components/MembershipInfoCard/index.jsx [hunk 2]
```diff
@@ -1,34 +1,85 @@
...
     }
   };
 
+  return (
+    <Grid container item xs={4} alignItems="center">
+      <Grid item xs={12} align="center">
+        <Switch
+          onChange={() => {
+            toggleMembershipPrivacy(element);
+          }}
```
Comment (amos-cha) [inline src/components/Profile/components/MembershipsList/components/MembershipInfoCard/index.jsx:46]:
```suggestion
          onChange={() => 
            toggleMembershipPrivacy(element);
          }
```

File: src/services/membership.ts [hunk 3]
```diff
@@ -117,6 +118,56 @@ const getFollowersNum = (involvementCode: string, sessionCode: string): Promise<
...
+  // GroupAdmin: boolean;
+  // IDNumber: number;
+  // LastName: string;
+  // MembershipID: number;
+  // Participation: Participation;
+  // ParticipationDescription: ParticipationDesc;
+  // Privacy: boolean | null;
+  // SessionCode: string;
+  // SessionDescription: string;
+  // StartDate: string;
```
Comment (amos-cha) [inline src/services/membership.ts:145]:
```suggestion
  // Description: string;
  // EndDate: string;
  // FirstName: string;
  // GroupAdmin: boolean;
  // IDNumber: number;
  // LastName: string;
  // MembershipID: number;
  // Participation: Participation;
  // ParticipationDescription: ParticipationDesc;
  // Privacy: boolean | null;
  // SessionCode: string;
  // SessionDescription: string;
  // StartDate: string;
```

remove if unused

**Review by amos-cha — CHANGES_REQUESTED** [review]
small cleanup then you're ready

**Review by amos-cha — APPROVED** [review]

**Review by amos-cha — APPROVED** [review]

**Pull request merged**